### DISCUSSION TITLE
Tidak dapat mengirim notifikasi

### DISCUSSION THREAD

**Discussion by mediasaranainformatika** [kirimwa-id/forum#108] [Tanya Jawab]
Halo mas rio,

Mau tanya mas, apa server kirimwa.id sedang down ya? Ada pesan error '...connected host has failed to respond'. Mohon konfirmasinya mas, terima kasih sebelumnya

**Comment by rioastamal** [comment]
@mediasaranainformatika Saat ini server API KirimWA.id masih menyala tidak ada masalah. Silahkan coba lagi.
